custom-namespaces.yaml

```
@@ -680,12 +680,29 @@ monitoring:
     - anicholson@mozilla.com
     - mhirose@mozilla.com
     - kignasiak@mozilla.com
+    - wichan@mozilla.com
   pretty_name: Monitoring (Data Pipeline)
   views:
     average_ping_sizes:
       type: ping_view
       tables:
         - table: mozdata.monitoring.average_ping_sizes
+    bigquery_tables_inventory:
+      tables:
+        - table: moz-fx-data-shared-prod.monitoring.bigquery_tables_inventory
```

Why not:
```suggestion
        - table: mozdata.monitoring.bigquery_tables_inventory
```
here and below?

---

This is to align with other monitoring datasets to be in the same project.